errors.go.tmpl

```diff
@@ -93,4 +93,28 @@ var (
 )
 {{ end -}}
 
+// Error mapping
+var (
+	ErrorsByName = map[string]*WebRPCError{
+{{- range $_, $error := $webrpcErrors}}
+		{{ printf "%q: &Err%s," $error.Name $error.Name }}
+{{- end}}
+{{- if len $schemaErrors }}
+{{- range $_, $error := $schemaErrors}}
+		{{ printf "%q: &Err%s," $error.Name $error.Name }}
+{{- end}}
+{{- end}}
+	}
+	ErrorsByCode = map[int]*WebRPCError{
```

Comment:
Can you please describe the high-level use cases for these maps?

What are you trying to achieve?

Isn't `ErrorsByCode` enough, why do we need `ErrorsByName` too?